templates/500.html

```diff
@@ -8,7 +8,16 @@
 <div class="p-strip is-deep">
   <div class="row">
     <div class="col-4">
-      <img src="https://assets.ubuntu.com/v1/4de1c56c-404_v1.svg" alt="Error owl" width="400" height="400" class="p-image--smaller-mobile"/>
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/4de1c56c-404_v1.svg",
+          alt="Error owl",
```

Comment:
Same, but maybe we should change also the image? The current image is for a 404

https://assets.ubuntu.com/v1/4de1c56c-404_v1.svg

```suggestion
          alt="Server error",
```

Reply:
Will talk to @danielmutis today about a new image, but for now I'll stick to this one as we use the same on canonical.com